lib/generators/framework/definition_generator.rb

```diff
@@ -0,0 +1,44 @@
+require 'csv'
+
+class Framework
+  ##
+  # Create a Framework::Definition derivative given a
+  # framework_short_name. See USAGE
+  class DefinitionGenerator < Rails::Generators::NamedBase
+    source_root File.expand_path('templates', __dir__)
+
+    desc 'This generator creates a framework definition file at app/models/framework_definitions'
```

Comment:
I think setting this overrides the `USAGE` file; it's either/or in terms of what the user sees in terms of help/usage.

Reply:
Didn't know this; I'll remove this `desc`. I prefer editing a `USAGE` as per above comment.

Reply:
Before/after removing `desc` (I like `USAGE` here):

```
➜  DataSubmissionServiceAPI git:(471-establish-framework-definitions-in-the-api) ✗ bin/rails g framework:definition --help
Running via Spring preloader in process 99257
Usage:
  rails generate framework:definition NAME [options]

Options:
  [--skip-namespace], [--no-skip-namespace]  # Skip namespace (affects only isolated applications)

Runtime options:
  -f, [--force]                    # Overwrite files that already exist
  -p, [--pretend], [--no-pretend]  # Run but do not make any changes
  -q, [--quiet], [--no-quiet]      # Suppress status output
  -s, [--skip], [--no-skip]        # Skip files that already exist

This generator creates a framework definition file at app/models/framework_definitions
➜  DataSubmissionServiceAPI git:(471-establish-framework-definitions-in-the-api) ✗ bin/rails g framework:definition --help
Running via Spring preloader in process 99326
Usage:
  rails generate framework:definition NAME [options]

Options:
  [--skip-namespace], [--no-skip-namespace]  # Skip namespace (affects only isolated applications)

Runtime options:
  -f, [--force]                    # Overwrite files that already exist
  -p, [--pretend], [--no-pretend]  # Run but do not make any changes
  -q, [--quiet], [--no-quiet]      # Suppress status output
  -s, [--skip], [--no-skip]        # Skip files that already exist

Description:
    Generates framework definitions from MISO CSV

Example:
    bin/rails g framework:definition "CM/OSG/05/3565"

    This will create:
        app/models/framework/definition/CM_OSG_05_3565.rb
```